src/components/Footer.jsx

```diff
@@ -1,7 +1,40 @@
+// import Image from "next/image";
+// import Logo from "@/public/Logo.webp";
+import { BsDiscord } from "react-icons/bs"; //  for discord
+import { BsInstagram } from "react-icons/bs"; //  for insta
+import { BsEnvelope } from "react-icons/bs"; // for mailing
+
 const Footer = () => {
   return (
-    <div>
-      <p>Footer</p>
+    <div className="box-content h-32 w-128 bg-gradient-to-r from-lime-950 via-lime-900 to-lime-800">
+      <p className="text-gray-100 absolute bottom-[-16%] left-[5%]">Tea & Go</p>
+
+      <div className="absolute bottom-[-19%] left-[1%]">
+        <img
+          className="h-16 max-w-xl rounded-lg shadow-xl dark:shadow-gray-800"
+          src="Logo.webp"
+        ></img>
```

Comment:
since there's nothing inbetween the image tags, you don't need a </img> tag. You can just to <img className="" src="" />. just add the / to the end of the tag and that will close it